src/compiler/crystal/codegen/link.cr

```diff
@@ -81,7 +81,7 @@ module Crystal
     end
 
     class_getter paths : Array(String) do
-      default_path.split(':', remove_empty: true)
+      default_path.split(Process::PATH_DELIMITER, remove_empty: true)
```

Comment:
Is there a reason to make `CRYSTAL_PATH`'s format platform-dependent?

Reply:
Well yeah, otherwise you literally can't specify any absolute path (C:)

Reply:
Oh, right...

:( platform-consistency